### ISSUE TITLE
TypedDjango now has financial contributors

### ISSUE THREAD

**Issue by sobolevn** [typeddjango/django-stubs#1398]
Right now we have 200$ in our OpenCollective account: https://opencollective.com/typeddjangoorg

I think it might be fair to split these funds among active maintainers.
So, I want to raise a discussion about it.

What do you think? @intgr @adamchainz @flaeppe 

(I cannot have my share due to the political stuff, because both OC and PayPal have blocked all transactions in my country. PayPal even stole my money I had on my account. So, I volunteer to share my money with other devs).

**Comment by intgr** [comment]
Awesome! Thanks to Pinecast for the sponsorship, and also individual contributors.

If there are some other expenses we could be making with these funds to help the project, I would happily contribute my share towards those. But I can't think of anything right now.

**Comment by flaeppe** [comment]
Thank you Pinecast for the donation.

I'd rather use my share to move the project forward too. In whatever way it could help.

Could it be worth to put a bounty on some issue(s)?

**Comment by sobolevn** [comment]
I don't have any experience with bounties. If anyone has, please share your feedback :)

**Comment by adamchainz** [comment]
I'm going to print some stickers of my book mascots for Djangocon Europe. Perhaps we could also do some django-stubs stickers?

**Comment by sobolevn** [comment]
@adamchainz sounds great! If others agree, you can send a payment request to https://opencollective.com/typeddjangoorg and I will approve it.

But, I am very bad at graphics: this is the best I can do https://avatars.githubusercontent.com/u/52535716?s=200&v=4 I won't be able to help here :(

P.S. I would be extra grateful to have one sticker by mail, but if it is too complex - I 100% understand :)

**Comment by intgr** [comment]
I had a look at the [Django Trademark License](https://www.djangoproject.com/trademarks/). Our remixing of the Django logo probably isn't allowed, this is the closest thing that only permits names of localities together with the logo:

> 3. Groups and Events
>
> [...]
> A group or event may use a logo that incorporates the Django logo by adding a word or picture that identifies the locality (for example, the name of a city, or a stylized image of an identifiable city landmark), audience, or activities of the group or event.

So we may need to redesign the logo anyway. If it's just a bunch of volunteers, this probably wouldn't be an issue, but now that real money has entered the picture, we should take it more seriously.

And this part is also relevant:

> 4. Merchandise
> 
> You may produce merchandise displaying the Django name or logo, provided that:
> 
> 1. Artwork to be printed on merchandise has been formally approved by the DSF;
> 2. The products on which the artwork is to appear has been formally approved by the DSF; and

**Comment by intgr** [comment]
Alternatively, if we get explicit approval from the DSF, then none of that matters.

**Comment by flaeppe** [comment]
> ... If others agree ...

I think it's a great idea

**Comment by adamchainz** [comment]
I would be pretty optimistic about getting DSF approval. I will send an email now.

I just tried putting together the below ideas for a project logo. I used the same font as the django logo, the colours from Mypy, and then some variants with icons from bootstrap to add some flavour.

![django-stubs logo ideas](https://user-images.githubusercontent.com/857609/235794049-69bb5021-0602-4454-b87b-5d613161cb2d.png)

Vote for your favourites or suggest other icons or edits!

**Comment by sobolevn** [comment]
`double-checked` is my first place 🏅

**Comment by intgr** [comment]
Thanks for making these. I also like "double-checked" the most.

> I would be pretty optimistic about getting DSF approval. I will send an email now.

Ah, never mind, I was talking about the current TypedDjango project icon <img src="https://user-images.githubusercontent.com/137616/235884024-597c6575-18ab-4cea-9b21-152895ffec17.png" width=20 />

The ones you have proposed are already allowed, since they don't use the Django "dj" logo. Using name "django-stubs" is allowed according to Trademark License point 2.

**Comment by adamchainz** [comment]
Alright, I’ve made #1499 with the new logo.

Here’s my proposed sticker design based on it, adding a 2mm white border:

![django-stubs logo sticker](https://github.com/typeddjango/django-stubs/assets/857609/29d510ef-444b-4fbc-b006-3597fac00036)

I've use https://stickershop.co.uk for my book cover stickers, to great success: https://twitter.com/AdamChainz/status/1655123942980349954

The stickershop [custom cut](https://www.stickershop.co.uk/products/custom-printed-die-cut-stickers) quote for 250 django-stubs stickers is £56 = $70. I don't think we'll need more than 250 for DjangoCon Europe.

Sound good?

**Comment by sobolevn** [comment]
Sounds great! 👍

**Comment by adamchainz** [comment]
Updated version with new proposed ticket shape from #1502.

![django-stubs logo sticker](https://github.com/typeddjango/django-stubs/assets/857609/a1b0323f-166a-4c1c-bc27-dab71078d4dd)

**Comment by intgr** [comment]
> proposed sticker design based on it, adding a 2mm white border

So the sticker is cut out at 2mm distance from the text?

I don't know anything about sticker printing. Could it be an issue that, because the sticker is narrowest around the `-` sign, that it becomes physically weak and might tear around that area?

**Comment by adamchainz** [comment]
Yes it’s a 2mm border. I surveyed the stickers on my laptop and 0.5-4mm seems to be the range. 2mm should give us enough strength.

Good point on potential weakness. The below https://django.social sticker was strong enough with a very similar design:

![IMG_4239](https://github.com/typeddjango/django-stubs/assets/857609/6ad2fddc-92bb-41d4-ac36-ad2e716c6b9d)

But it is a fair point. I will try a second design with more of a “bridge”….

**Comment by adamchainz** [comment]
![django-stubs logo sticker](https://github.com/typeddjango/django-stubs/assets/857609/97b6d53a-0e1e-4446-98b3-6fead7befc1f)

Looks fine to my eye, wdyt?

**Comment by intgr** [comment]
Yep, looks good even with a black background. If I didn't know you tweaked the bridge, I wouldn't notice it.

**Comment by adamchainz** [comment]
I forgot to report back here. I placed many successive batches of stickers on the sticker table at DjangoCon Europe and they were all taken. The remainders are with me and will be distributed monthly at [Django London](https://www.meetup.com/djangolondon/) 😁

I’ve just submitted [my expense on Open Collective](https://opencollective.com/typeddjangoorg/expenses/154575). @sobolevn can you approve, and add me as a team member? Ta!

**Comment by sobolevn** [comment]
Approved! Thanks a lot for doing this! 

I've also added you as a admin to https://opencollective.com/typeddjangoorg
You need to accept an invite.

**Comment by sobolevn** [comment]
[@getsentry](https://github.com/getsentry) is now sponsoring @typeddjango for $5.00 / month!

Thank you!

**Comment by sobolevn** [comment]
<img width="549" alt="Снимок экрана 2023-12-04 в 13 00 53" src="https://github.com/typeddjango/django-stubs/assets/4660275/7a00b545-2690-453b-b89f-8ce9c73a0003">

🎉

**Comment by sobolevn** [comment]
We also have Sentry as a supporter on thanks.dev
I hope that I made everything right and this payout will be seen on our github. I didn't enter any other payment details, so I think that this is the only logical way. Will see :)

<img width="974" alt="Снимок экрана 2024-01-29 в 11 59 00" src="https://github.com/typeddjango/django-stubs/assets/4660275/975026cb-7423-4708-89c1-72ffc27ede4e">

**Comment by sobolevn** [comment]
Btw, our GitHub funds are not transfered to OpenCollective funds. I think that I won't be able to setup the proper redirect anytime soon :)

**Comment by sobolevn** [comment]
It worked:
<img width="737" alt="Снимок экрана 2024-01-31 в 09 50 50" src="https://github.com/typeddjango/django-stubs/assets/4660275/68f8b791-8b71-45f0-bb83-14fd79519a65">

**Comment by sobolevn** [comment]
<img width="614" alt="Снимок экрана 2024-03-19 в 14 01 41" src="https://github.com/typeddjango/django-stubs/assets/4660275/eaa3245a-bed0-41c6-bea8-bdc7dc9691af">

Thank you!

**Comment by sobolevn** [comment]
Hey, everyone! We currently have:
- 1600$ on https://opencollective.com/typeddjangoorg
- 100$ on https://thanks.dev, I transfered it to OpenCollective just now
- 560$ on https://github.com/sponsors/typeddjango/dashboard (I am working on sending them to OpenCollective right now)

All in all we have around 2260$ (I say around, because the final amount is smaller and there will be large transaction fees)

I propose to take 500$ bonus for each of the active maintainers: me, @intgr, @flaeppe, @adamchainz 

There's no reason for these funds just to be there with no use.

*Edit*: done, money from GitHub sponsors are transfered to OC! 🎉 
<img width="460" alt="Снимок экрана 2024-07-11 в 10 57 07" src="https://github.com/typeddjango/django-stubs/assets/4660275/3772cb14-5dd6-4b07-a38c-c4e7e07bda38">

**Comment by flaeppe** [comment]
Alright, fair enough. Any suggestion for the most practical way to move any amount?

**Comment by sobolevn** [comment]
You can send a spending request on OpenCollective with "support and development" title. I will approve it. In this request you will specify all the info that OC needs to send you your money. It is rather simple. @adamchainz already did this once.

**Comment by sobolevn** [comment]
@flaeppe I approved.

One more note, on my behalf https://github.com/proofit404 will receive my part, because in 2022 PayPal blocked my account and stole my money. So, I cannot use OC directly now.

**Comment by flaeppe** [comment]
Thank you very much!

**Comment by adamchainz** [comment]
Thank you for managing this @sobolevn , splitting the funds is fair.

I feel a bit guilty that I haven't been so active on django-stubs recently. I *have* been promoting django-stubs by distributing the stickers at DjangoCon Europe and London Django :) Also, we didn’t get a GSoC candidate this year, so I didn't have anyone to mentor there. I will be sure to return to helping more on django-stubs soon though...

**Comment by sobolevn** [comment]
@adamchainz approved 👍

**Comment by flaeppe** [comment]
@sobolevn OC wanted me to edit the expense to include a timeframe. So now it needs reapproval, if you wouldn't mind 🙂

**Comment by sobolevn** [comment]
This is my friend getting my check for me:
<img width="743" alt="Снимок экрана 2024-08-14 в 11 21 06" src="https://github.com/user-attachments/assets/be99920f-acc9-41e4-99f4-1776d059af8c">

**Comment by adamchainz** [comment]
@sobolevn there's an outstanding comment from August 11th - the invoice details need updating.

**Comment by sobolevn** [comment]
Very big "Thank you" goes to https://funds.ecosyste.ms/funds/django
Who donated 300+ $ to TypedDjango! This is awesome 🎉 

<img width="583" alt="Снимок экрана 2025-01-13 в 18 17 58" src="https://github.com/user-attachments/assets/f8a653d9-35e9-45d1-936f-909008fe9eb6" />

Congrats to all of the team for their hardwork. Well earned! 👏
